content/en-us/chat/customizing-in-experience-text-chat.md

```diff
@@ -207,6 +207,15 @@ end
 TextChatService.OnIncomingMessage = onIncomingChatMessage
 ```
 
+## Adding A System Message
+
+By using [:DisplaySystemMessage](https://create.roblox.com/docs/reference/engine/classes/TextChannel#DisplaySystemMessage) you are able to create a system-like message. This can be used to greet users, or alert them when an ingame event is happening.
```

Comment:
```suggestion
By calling `Class.TextChannel:DisplaySystemMessage()`, you can display a system-like message, useful for greeting users or alerting them when an in-experience event is happening.
```

Reply:
For future reference on linking within docs site, see:
https://github.com/Roblox/creator-docs/blob/main/STYLE.md#links